### ISSUE TITLE
[Feature Request] v3: impossible to know which of the segmentation series is being displayed

### ISSUE THREAD

**Issue by fedorov** [OHIF/Viewers#3421]
### What feature or change would you like to see made?

Indicate to the user which segmentation series is being displayed

### Why should we prioritize this feature?

There may be more than one segmentation series in the study. In v2, it was possible to access individual segmentations in the UI, but in v3 I do not see any UI elements to indicate which series is being shown. For a demonstration and to see the differences between v2 and v3, please see #3420 that has links and screenshots.

**Comment by 03shek** [comment]
I think it's because after loading segmentations with "YES", the SEG names aren't auto-populating. The user must manually rename them again.
![image](https://github.com/OHIF/Viewers/assets/115667958/6558b78b-1248-4d30-bf34-78fca40991a3)

**Comment by jbocce** [comment]
@fedorov, would you consider #3426 covering this request?

**Comment by fedorov** [comment]
I have hard times understanding what exactly is requested in that issue.

**Comment by 03shek** [comment]
@fedorov If there's a name for a SEG or RTSTRUCT file on the left, when that file is loaded, that name should be present in the collapsible SEG list above its ROIs

**Comment by fedorov** [comment]
Thank you for the clarification! I am not sure what "name" means, but showing SeriesDescription for the loaded SEG/RTSTRUCT would do it for me.

**Comment by jbocce** [comment]
Duplicate of #3426.

**Comment by jbocce** [comment]
Hi @fedorov, here is a [PR](https://github.com/OHIF/Viewers/pull/3517) that was just merged, could you please have a look to determine if it addresses this feature request? Thanks.

**Comment by fedorov** [comment]
Can you please confirm if it is live in https://viewer.ohif.org/?

**Comment by fedorov** [comment]
I'd say the current implementation is not sufficient. Neither SeriesDescription, nor SeriesNumber are required to have unique values. Would it be possible to highlight the corresponding series in the list on the left?

**Comment by fedorov** [comment]
Also, what is the semantics of that name shown on the right? If it is supposed to match SeriesDescription, why is it editable? Are you planning to overwrite SeriesDescription at the source if the user renames it? This would not be a good idea.

**Comment by 03shek** [comment]
I agree. Those editable features must've been carried over from the measurements list. In general, RTstructs and SEGs are not overwritten anyway, but created anew. In the context of OHIF, these files are read-only. On the left, it currently doesn't show any series numbers for RTstructs or SEGs. If there could be a highlighted bounding box to also indicate a SEG has already been loaded in the session, that would also be helpful.

**Comment by 03shek** [comment]
And yes, in essence, your recommendation above is identical to #3357. Since there's no way to create SEGs or overwrite them in Basic Viewer at present, relabeling them would achieve nothing outside of the current study session.

**Comment by 03shek** [comment]
Thanks @jbocce . Both segment labels and series description on the right side should NOT be editable.

**Comment by jbocce** [comment]
Thanks @03shek. Just to further clarify, are you suggesting that ALL series descriptions and labels circled in red below should NOT be editable?

![image](https://github.com/OHIF/Viewers/assets/109477394/e145afab-7064-4e34-a2c3-1d7b34b9fba7)

**Comment by fedorov** [comment]
> are you suggesting that ALL series descriptions and labels circled in red below should NOT be editable?

I agree with @03shek - I have yet to hear the explanation why they would be editable, and where those edited items will be saved.

> in essence, your recommendation above is identical to https://github.com/OHIF/Viewers/issues/3357.

@03shek I think it is different. The issue here is that certain items should not be editable at all (in this case, modification of SeriesDescription for the existing series or modification of anything inside that series, unless the user is clearly alerted what happens after this edit is done and where the modified content is saved), under any circumstances. The issue in #3357 is about disabling the ability to edit metadata or the image annotations. Do you agree?

**Comment by jbocce** [comment]
Just to be clear, at this point I want to gather what is required. I am not trying to explain whether they should or should not be editable. Once the requirements are gathered then we can go from there. Does that make sense? 😊

**Comment by fedorov** [comment]
I am trying to understand the logic of the current behavior, since that might change my perspective on the requirements. But fair enough - I hope I provided sufficient justification for the expected behavior in the above - let me know if you need any more clarifications!

**Comment by 03shek** [comment]
@fedorov Right. #3357 is not a redundant issue. But one of the 3 bullets: 

> Is it possible to disable all editing features (both editing of the images, but also editing of things like segment labels)?

The last bit is relevant to the current issue @3421 as an updated requirement; to disable editing segment labels (in Basic Viewer only). There's no point in modifying SEGs in Basic Viewer if they were never able to be created or saved from there in the first place. 

I think SEGs can only be created in TMTV mode. It makes sense, there, to leave those segment labels and Series Description editable.

**Comment by dan-rukas** [comment]
Here are some recommendations for a color state change in the Studies panel:

* "list icon" - color #5ACCE6
* "SEG" label - background color: #5ACCE6, text color: #000000 (black)

<img width="505" alt="SEG-color-state" src="https://github.com/OHIF/Viewers/assets/2745168/9c9eb6db-d22f-4f48-97ef-095607ea7599">

**Comment by james-hanks** [comment]
See #3426

**Comment by igoroctaviano** [comment]
@dan-rukas any updates on this one?

**Comment by jbocce** [comment]
@igoroctaviano, there is a merged PR #3594 that addresses this.

**Comment by james-hanks** [comment]
Reopening to address feedback about unaddressed issues:
- highlight seg in study panel
- link icon in study panel
- indicator for which study seg is derived from

**Comment by jenny-hm-lee** [comment]
Hi @james-hanks , @dan-rukas 
Any update on this issue and its related issue #3790 ?

**Comment by sedghi** [comment]
Yes we have a design, we will work towards implementation

**Comment by jenny-hm-lee** [comment]
Thanks @sedghi , 
Do you have an estimate timeline for this and #3790 ?

**Comment by sedghi** [comment]
Unfortunately, no.

**Comment by sedghi** [comment]
This has been merged, check it out in master, and viewer-dev

**Comment by sedghi** [comment]
We just release the OHIF 3.8, you can find more details here https://ohif.org/release-notes/3p8/
If you still encounter this issue in 3.8, please re-open this.
